overlord/devicestate/handlers_install.go

```diff
@@ -1010,6 +1012,30 @@ func (m *DeviceManager) doInstallFinish(t *state.Task, _ *tomb.Tomb) error {
 	}
 	defer unmountParts()
 
+	if !sys.Model.Classic() {
+		sysSeed, err := seedOpen(dirs.SnapSeedDir, sys.Label)
```

Comment:
double-checking, we don't open a seed elsewhere in this function?

Reply:
we do, but AFAITC it's not directly returned from `loadAndMountSystemLabelSnaps()`, perhaps a little refactor would do it

Reply:
Moved things around so we only open it once now.